.github/workflows/telegram-release.yml

```
@@ -104,20 +119,30 @@ jobs:
+          TELEGRAM_NOTES_PATH: ${{ steps.version_info.outputs.telegram_notes_path }}
         run: |
           STORE_APK="app/build/distribution/store/store-release.apk"
           FOSS_APK="app/build/distribution/foss/foss-release.apk"
           
           # 1. Construct the Caption
           CAPTION="⚡️ *Thor v${{ env.VERSION_NAME }} Released!*
+
+Build: ${{ env.VERSION_CODE }}
+Branch: ${{ github.ref_name }}
+
+"
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)